Cakes-Co/css/style.css

```diff
@@ -0,0 +1,312 @@
+/*  Add your styling here */
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --pink: rgb(242 200 217);
+  --darker-pink: rgb(201, 40, 105);
+  --light-pink: rgb(249 237 244);
+  --background-color: var(--white);
+  --gold: rgb(160 150 111);
+  --white: rgb(255, 255, 255);
+
+  --page-width: 100vw;
+
+  --default-font-family: "Nunito Sans", sans-serif;
+  --heading-font-family: "Playfair Display", serif;
+
+  font-size: 16px;
+}
+
+body {
+  background-color: var(--background-color);
+  color: var(--darker-pink);
+  font-family: var(--default-font-family);
+}
+
+.container {
+  margin: 0 auto;
+  position: relative;
+}
+
+a {
+  text-decoration: none;
+  color: var(--gold);
+  transition: all 0.2s ease-in-out;
```

Comment:
I'm not quite sure why you've added this transition, doesn't seem to be doing anything in the page from my point of view. Could you tell me why?

Reply:
It's making a transition between 2 "a" elements in the nav in transition.